test/classes/initializers/generics/init-param-from-non-param.chpl

```diff
@@ -0,0 +1,13 @@
+config var notaparam = 77;
```

Comment:
Not a bad thing, but I'm curious why you went with this program as opposed to using the local var you mentioned in the original issue

Reply:
In the original issue, I was hung up on the ``record`` type, and the local var was just there to separate the param initialization from the ``new R()``.  When I found the issue applied to ``int`` as well, I used a ``config var`` so I didn't have to contemplate whether the compiler believes the value below is "known at compile time" as the error message says:

```chapel
var notaparam = 7;
param p = notaparam;
```

With a config var, there's no sense in which it's known at compile time.

Reply:
I can see that logic, but a local variable is a more basic case (assuming it still causes the same failure) and really should be treated reasonably similarly by the compiler.

Reply:
Ok.  I'll try making that change tonight.